.github/ISSUE_TEMPLATE/F-submit-statistical-software-for-review.md

```diff
@@ -77,4 +77,4 @@ This package:
 
 ## Code of conduct
 
-- [ ] I agree to abide by [rOpenSci's Code of Conduct](https://ropensci.github.io/dev_guide/policies.html#code-of-conduct) during the review process and in maintaining my package should it be accepted.
+- [ ] I agree to abide by [rOpenSci's Code of Conduct](https://devguide.ropensci.org/policies.html#code-of-conduct) during the review process and in maintaining my package should it be accepted.
```

Comment:
why link to this and not the Code of Conduct on rOpenSci website directly?

Reply:
Yes, i was wondering that myself. That line is taken directly from main submission template (A). I'll update both of them right now ...